plugins/scaffolder/src/components/TaskPage/TaskPage.tsx

```diff
@@ -266,6 +273,26 @@ export const TaskPage = () => {
 
   const { output } = taskStream;
 
+  const handleStartOver = () => {
+    if (!taskStream.task || !taskStream.task?.spec.metadata?.name) {
+      navigate(generatePath(rootLink()));
+    }
+
+    const formData =
+      taskStream.task!.spec.apiVersion === 'backstage.io/v1beta2'
+        ? taskStream.task!.spec.values
+        : taskStream.task!.spec.parameters;
+
+    navigate(
+      generatePath(
```

Comment:
@Rugvip little bit of a side question here after the `routeRef` deprecations and us trying to remove usages of `generatePath`, are we thinking that we should build the URL ourself, keep this usage, or should be be creating `subRouteRefs` instead for the scaffolder for the different pages with params in the URLs instead?

Reply:
I think a SubRouteRef would be the way to go here, but can leave that for a later change imo